master/security/service-accounts.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Use service accounts in policy rules
+title: Use service accounts in policy
 description: Use Kubernetes service accounts in policy rules to validate cryptographic identities and/or manage RBAC controlled high-priority rules across teams.
```

Comment:
"policy rules" -> "policy"

Reply:
I think we need to update this line too: https://github.com/projectcalico/calico/pull/2911/files#diff-ad42278cb872bb083fb60cb21c746d85R22

Reply:
Fixed

Reply:
> Use Kubernetes service accounts in policy rules to validate cryptographic identities and/or manage RBAC controlled high-priority rules across teams.

Still says "rules" here